src/commands/deploy.rs

```diff
@@ -200,11 +200,15 @@ impl DeployCommand {
 
     async fn compute_buildinfo(&self, cfg: &RawAppManifest) -> Result<BuildMetadata> {
         let mut sha256 = Sha256::new();
+        let mut app_folder = PathBuf::from(&self.app);
+        app_folder.pop();
```

Comment:
Super nit: elsewhere we have used `parent()` for this to avoid the two-step and to pick up edge case errors (e.g. `let app_folder = &self.app.parent().with_context(...)?;`.

Reply:
Updated it. Thank you.